packages/formik/src/Formik.tsx

```diff
@@ -81,6 +84,7 @@ function formikReducer<Values>(
     case 'SET_FIELD_TOUCHED':
       return {
         ...state,
+        focus: undefined,
```

Comment:
I wouldn't expect `setFieldTouched` to remove focus in all scenarios. For example, if a user is entering into an input and I programatically set the field to touched so the error displays.

Reply:
Fair. Thus the onBlur/handleBlur will set focus then.